language/documentation/tutorial/README.md

```diff
@@ -219,9 +219,10 @@ to [`step_2/BasicCoin`](./step_2/BasicCoin).  Unit tests in Move are similar to
 unit tests in Rust if you're familiar with them -- tests are annotated with
 `#[test]` and written like normal Move functions.
 
-You can run the tests with the `package test` command:
+You can build the package and run the tests with the `package test` command:
 
 ```bash
+move package build
```

Comment:
I think the `package test` command will auto do the `build`, is need to do it before?

Reply:
Ok, after some more testing, I think my original description is wrong, though there is still a problem.

When you run `move package test` the first time, it is fine.

However, if you run `move package test` again, it fails:

> RUST_BACKTRACE=1 move package test
CACHED MoveStdlib
BUILDING BasicCoin
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: No such file or directory './build/MoveStdlib/sources/ASCIITests.move'', language/tools/move-cli/src/package/cli.rs:510:83
stack backtrace:
   0: rust_begin_unwind
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/core/src/panicking.rs:143:14
   2: core::result::unwrap_failed
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/core/src/result.rs:1749:5
   3: core::ops::function::impls::<impl core::ops::function::FnMut<A> for &mut F>::call_mut
   4: move_package::compilation::compiled_package::CompiledPackage::build
   5: move_package::compilation::build_plan::BuildPlan::compile_with_driver
   6: move_cli::package::cli::run_move_unit_tests
   7: move_cli::package::cli::handle_package_commands
   8: move_cli::run_cli
   9: move_cli::move_cli
  10: move::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

If you run `move package build` and then `move package test`, it is good. That's why I came up with this PR. But now I guess it's not actually the solution.

Reply:
@ShuaoZhang It sounds like a bug of `move pakcage`, please help to submit a bug report for this issue, and I think this document improvement also needed.

Reply:
#161 Thanks.